composer.json

```diff
@@ -37,6 +37,7 @@
         "typo3/cms-form": "^9.5",
         "typo3/cms-impexp": "^9.5",
         "typo3/cms-info": "^9.5",
+        "typo3/cms-lowlevel": "^9.5",
```

Comment:
One final thing would be to run `ddev composer update` to update `content-hash` in `composer.json`.